src/QuickMarcEditor/QuickMarcCreateWrapper.js

```diff
@@ -39,6 +39,7 @@ const propTypes = {
   location: ReactRouterPropTypes.location.isRequired,
   locations: PropTypes.object.isRequired,
   marcType: PropTypes.oneOf(Object.values(MARC_TYPES)).isRequired,
+  marcSpec: PropTypes.object.isRequired,
```

Comment:
I think `marcSpec` prop name is a little confusing. Without knowing what it is I though it was a general MARC file specification, but it's just for Fixed Fields. Let's rename all it's uses to `fixedFieldSpec`

Reply:
Yes I had doubt too with this name